src/main/java/roboy/util/ConfigManager.java

```diff
@@ -107,6 +110,9 @@ private static void initializeConfig() {
             CONTEXT_GUI_ENABLED = yamlConfig.getBoolean("CONTEXT_GUI_ENABLED");
 
             ACTION_CLIENT_SCRIPT = yamlConfig.getString("ACTION_CLIENT_SCRIPT");
+
+            START_MEMORY_MODULE = yamlConfig.getString("START_MEMORY_MODULE").equals("true");
```

Comment:
Why not use getBoolean here? If you use getString on purpose, better to compare with equalsIgnoreCase instead of equals